gulpfile.js

```diff
@@ -109,6 +109,16 @@ gulp.task('copy-fonts', () => {
   return gulp.src('src/fonts/**/*').pipe(gulp.dest('dist/fonts/'));
 });
 
+// Copies content default locale into it's own folder on prod.
+gulp.task('default-locale', () => {
+  if (isProd) {
+    return gulp
+      .src(['./dist/**/*.html', '!./dist/i18n/**/*.html'])
```

Comment:
This copies the html files but not the images. If firebase were to (for whatever reason) use this directory would the relative image paths all break?

Reply:
No, images would be fine. The top level is the default and will fallback to that when no language is provided.